vmhost/vmhooks/manBufOps.go

```diff
@@ -686,11 +686,15 @@ func v1_4_mBufferStorageLoad(context unsafe.Pointer, keyHandle int32, destinatio
 		return 1
 	}
 
-	storageBytes, usedCache, err := storage.GetStorage(key)
+	storageBytes, trieDepth, usedCache, err := storage.GetStorage(key)
 	if vmhost.WithFault(err, context, runtime.ManagedBufferAPIErrorShouldFailExecution()) {
 		return 0
 	}
-	storage.UseGasForStorageLoad(mBufferStorageLoadName, metering.GasSchedule().ManagedBufferAPICost.MBufferStorageLoad, usedCache)
+
+	err = storage.UseGasForStorageLoad(mBufferStorageLoadName, int64(trieDepth), metering.GasSchedule().ManagedBufferAPICost.MBufferStorageLoad, usedCache)
+	if vmhost.WithFault(err, context, runtime.ManagedBufferAPIErrorShouldFailExecution()) {
+		return 1
```

Comment:
1? Should it be 0 or -1? Should have used constants not magic numbers.

Reply:
@camilbancioiu which one should be here?

Reply:
`0`, for consistency with the other returns of this function.

In VM v1.5, these return values will be redone.

Reply:
Done.